docs/components/dartpad/dividers/theme/main.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: _buildShrineTheme(),
+      home: Scaffold(
+        body: DividerThemeDemo(),
+      ),
+    );
+  }
+}
+
+class DividerThemeDemo extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: <Widget>[
+          Expanded(
+            child: Container(
+              color: Colors.amber,
```

Comment:
I'd suggest removing the colors, since it makes it harder to see the divider itself. And then regenerate the screenshots.

Reply:
Done